vendor/knative.dev/pkg/version/version.go

```diff
@@ -33,7 +33,7 @@ const (
 	// NOTE: If you are changing this line, please also update the minimum kubernetes
 	// version listed here:
 	// https://github.com/knative/docs/blob/mkdocs/docs/snippets/prerequisites.md
-	defaultMinimumVersion = "v1.19.0"
+	defaultMinimumVersion = "v1.20.0"
```

Comment:
I'm not sure why this is updated.

/hold

Reply:
... and if we want this here.

Reply:
Please do not do this

Reply:
here is some background: https://coreos.slack.com/archives/CD87JDUB0/p1638183172256100

Reply:
I know, that's the reason I put a hold.

Reply:
yeah we want this to be 1.19